portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase

```diff
@@ -2107,6 +2107,78 @@ definition {
 		}
 	}
 
+	@description = "LPS-153512 - Test Automation for LPS-94855: I can edit only the first created segment"
+	@priority = "4"
+	test CanEditSecondSegmentCreated {
+		task ("Create first segmentation") {
+			Segmentation.openSegmentsAdmin(siteURLKey = "guest");
+
+			LexiconEntry.gotoAdd();
```

Comment:
Hey @rafaellajordao! In this case, you could create those Segments using JSON, and then just navigate to Segment Admin to edit them.